<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStackedWidget#stackedWidget
{  
	border-image: url(:/images/bk.png);
}

QStackedWidget#stackedWidget QPushButton
{
	border:none;
	border-radius:5px;
	background-color: rgb(11,194,255);
	color:white;
	font:normal 20pt &quot;微软雅黑&quot;;
}

QStackedWidget#stackedWidget QPushButton:hover
{
	background-color: rgb(0,182,250);
}

QLabel
{
	color:white;
	font:normal 30pt &quot;微软雅黑&quot;;
}



</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="menuPage">
      <widget class="QPushButton" name="lookBtn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>260</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>查看学生</string>
       </property>
      </widget>
      <widget class="QPushButton" name="statBtn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>310</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>统计学生</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quitBtn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>360</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>退出系统</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>341</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>学生成绩管理系统</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="lookPage">
      <widget class="QTableWidget" name="lookTableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>781</width>
         <height>551</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addBtn">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>80</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>添加学生</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveBtn">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>保存修改</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="statPage">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>310</y>
         <width>621</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string>statPage</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
